### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by GoldenSpade**
[Done](https://github.com/BilliZane/js_task-transportation-on-vacation/blob/develop/src/calculateRentalCost.js)

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  let discount = 0;
+
+  if (days >= 7) {
+    discount = 50;
```
Comment (illisogori) [inline src/calculateRentalCost.js:35]: можно вынести в переменную.

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,16 @@
...
+  } else if (days >= 3) {
+    discount = 20;
```
Comment (illisogori) [inline src/calculateRentalCost.js:37]: можно вынести в переменную.

**Review by illisogori — APPROVED** [review]
Отлично!

**Review by VinnieJ-2k20 — APPROVED** [review]
I approve, because it works correctly.

But for the future, Igor is right. 
Discounts are better used in variables here.
That shows the logic of your calculations much more clearly.

**Pull request closed without merging**